PAS_Database/dbo/Stored Procedures/Procs2/VendorPaymentList.sql

```diff
@@ -28,6 +28,7 @@
 	12   02/11/2023   Devendra Shekh  Changes for nonpo details
 	13   15/11/2023   Moin Bloch      added DueDate and Days Past Due
 	14   07/03/2024   AMIT GHEDIYA    Update Status Name as per PN-6767 (Filter param added)
+	15   26/03/2024   AMIT GHEDIYA    Update to get partial payment also show in Print Check
```

Comment:
@amitghediyasilverxis , Devendra has changes this SP can you get latest of this SP and Raise a pull request again.

Reply:
Okay, sure